referenceframe/frame_system.go

```diff
@@ -317,58 +316,37 @@ func (sfs *simpleFrameSystem) FrameSystemSubset(newRoot Frame) (FrameSystem, err
 // the point representation of their geometry type with respect to the world.
 func FrameSystemToPCD(system FrameSystem, inputs map[string][]Input, logger golog.Logger) (map[string][]r3.Vector, error) {
 	vectorMap := make(map[string][]r3.Vector)
-	geoMap, err := FrameSystemGeometries(system, inputs, logger)
+	geometries, err := FrameSystemGeometries(system, inputs)
 	if err != nil {
-		return nil, err
+		logger.Debug(err)
```

Comment:
why do we debug and not just return err

Reply:
The original logic involved logging an error and I was trying to preserve that